tests/core/service/health/health_service_test.py

```diff
@@ -0,0 +1,90 @@
+from unittest.mock import MagicMock, AsyncMock
+import pytest
+
+from symphony.bdk.gen import ApiException
+from symphony.bdk.gen.agent_model.agent_info import AgentInfo
+from tests.utils.resource_utils import object_from_json_relative_path
+
+from symphony.bdk.core.service.health.health_service import HealthService
+
+from symphony.bdk.gen.agent_api.system_api import SystemApi
+from symphony.bdk.gen.agent_api.signals_api import SignalsApi
+from symphony.bdk.gen.agent_model.v3_health import V3Health
+
+
+@pytest.fixture()
+def mocked_system_api_client():
+    api_client = MagicMock(SystemApi)
+    api_client.v3_health = AsyncMock()
+    api_client.v3_extended_health = AsyncMock()
+    return api_client
+
+
+@pytest.fixture()
+def mocked_signals_api_client():
+    api_client = MagicMock(SignalsApi)
+    api_client.v1_info_get = AsyncMock()
+    return api_client
+
+
+@pytest.fixture()
+def health_service(mocked_system_api_client, mocked_signals_api_client):
+    return HealthService(mocked_system_api_client, mocked_signals_api_client)
+
+
+@pytest.mark.asyncio
+async def test_health_check(health_service, mocked_system_api_client):
+    mocked_system_api_client.v3_health.return_value = object_from_json_relative_path(
+        "health_response/health_check.json")
+
+    health: V3Health = await health_service.health_check()
```

Comment:
is that necessary to put the type here? isn't here inferred from the method's return type?

Reply:
yes. I will remove it and all type annotations like these in the file.